packages/server/src/routes/auth/register.ts

```
@@ -83,13 +84,22 @@ export const registerHandler: AppRouteHandler<RegisterRoute> = async (c) => {
 		phone,
+		agreeTerms,
 	});
 	if (!newAuthUser) return c.json({ error: "User creation failed" }, 400);
 
 	const [newClient, error] = await Client.createNewClient({
 		authId: newAuthUser.id,
 	});
 
+	const business = await Business.createBusiness({
+		name: name,
+		clientId: newClient!.id,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)